users/klackygears/ReadMe.md

```diff
@@ -0,0 +1,134 @@
+Copyright <2019> <James Smith> @<klackygears>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+
+
+#<><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><><>#
+`---------------------------------------------------------------------------`
+Flash dfu bootloader on pro micro with arduino uno using adruino as ISP.
```

Comment:
This is in the ISP docs: https://docs.qmk.fm/#/isp_flashing_guide

Reply:
To be honest, I forgot this was in there. I just wanted to document my steps so I wouldn't forget! :-) Mostly, I wanted to keep the images of the uno wired to the pro micro. I'll save this somewhere else!